modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/SidebarPanel.js

```diff
@@ -12,28 +12,96 @@
  * details.
  */
 
+import ClayAlert from '@clayui/alert';
 import ClayLoadingIndicator from '@clayui/loading-indicator';
-import React, {useImperativeHandle, useState} from 'react';
+import {fetch} from 'frontend-js-web';
+import React, {
+	useCallback,
+	useEffect,
+	useImperativeHandle,
+	useRef,
+	useState,
+} from 'react';
 
 import Sidebar from './components/Sidebar';
 
-const SidebarPanel = React.forwardRef(({onClose}, ref) => {
+const SidebarPanel = React.forwardRef(({View, fetchURL, onClose}, ref) => {
```

Comment:
I think it might be a little odd to have a prop starting with a capital letter. I would opt for lowercase and then rename as a capital so that it can be used as a jsx element.

```js
{viewComponent: View, fetchURL, onClose}
```

Reply:
Yep, definitely makes more sense!